solidity/ERC721_WEB3.md

```diff
@@ -0,0 +1,183 @@
+## Solidity Contract with cryptozombie
+
+- **TokenFactory**
+
+  This contract generates random Tokens.
+
+  > ```
+  > contract TokenFactory is Ownable {
+  >     // event
```

Comment:
How about adding the comment like
`/*** EVENT ***/` instead of `// event`, 
`/*** DATA TYPES ***/` before `struct Token {...}`, 
`/*** STORAGE ***/` before `mapping (uint => address) public tokenToOwner;`?

I think these makes your example code more clear to understand!

Reply:
Do you mean to use comment /*** ***/ instead of using // in every part of the file?